script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_2_2.txt

```diff
@@ -2,22 +2,22 @@
 "........."
 ...I already know.
 If I want to return to a normal life as a human being, I should just run away.
-I'm fine with that. For me, that's simply the best choice.
-But,
+I'm fine with that. For me personally, that's simply the best choice.
```

Comment:
might just be the tone with which I read this internally, but I still like the original more here.